src/Config.hs

```diff
@@ -58,21 +73,31 @@ data Environment
     deriving (Eq, Show, Read)
 
 -- | This returns a 'Middleware' based on the environment that we're in.
-setLogger :: Environment -> Middleware
-setLogger Test = id
-setLogger Development = logStdoutDev
-setLogger Production = logStdout
+setLogger :: Environment -> LogEnv -> Middleware
+setLogger Test _          = id
+setLogger Development env = katipLogger env
+setLogger Production env  = katipLogger env
+
+-- | Web request logger (currently unimplemented). For inspiration see
+-- ApacheLogger from wai-logger package.
+katipLogger :: LogEnv -> Middleware
+katipLogger env app req respond = runKatipT env $ do
+    -- todo: log proper request data
+    logMsg "web" InfoS "todo: received some request"
+    liftIO $ app req respond
 
 -- | This function creates a 'ConnectionPool' for the given environment.
 -- For 'Development' and 'Test' environments, we use a stock and highly
 -- insecure connection string. The 'Production' environment acquires the
 -- information from environment variables that are set by the keter
 -- deployment application.
 makePool :: Environment -> IO ConnectionPool
-makePool Test =
-    runNoLoggingT (createPostgresqlPool (connStr "-test") (envPool Test))
-makePool Development =
-    runStdoutLoggingT (createPostgresqlPool (connStr "") (envPool Development))
+makePool Test = do
+    env <- mkLogEnv
```

Comment:
Rather than making a throwaway log pool, it seems like passing the parameter in would be cleaner.